### ISSUE TITLE
Show error on `repr` attribute without parenthesis

### ISSUE THREAD

**Issue by Undin** [intellij-rust/intellij-rust#6336]
## Environment

* **IntelliJ Rust plugin version:** 0.3.135.3449-202-nightly
* **Rust toolchain version:** 1.49.0-nightly (ffa2e7ae8 2020-10-24) x86_64-apple-darwin
* **IDE name and version:** IntelliJ IDEA 2020.2.3 Ultimate Edition (IU-202.7660.26)
* **Operating system:** macOS 10.15.6

## Problem description
Currently, the plugin doesn't produce error annotation on `repr` attribute without parenthesis although `rustc` produces "error: malformed `repr` attribute input" error

![image](https://user-images.githubusercontent.com/2539310/97812413-37afc080-1c92-11eb-8d83-ba3f35108c55.png)

## Steps to reproduce
```rust
#[repr]
pub enum Color {
    Black = 0
}

```

**Comment by Undin** [comment]
See [org.rust.ide.annotator.RsErrorAnnotator#checkReprAttribute](https://github.com/intellij-rust/intellij-rust/blob/bcd253717385d0e6a8e8300c05dd85478c0bb253/src/main/kotlin/org/rust/ide/annotator/RsErrorAnnotator.kt#L907) method

**Comment by zeroeightysix** [comment]
I'd like to try fixing this.
